### PR TITLE
Climate projection

### PR DESCRIPTION
**Pull request by tung-nd**
Code for finetuning ClimaX for climate projection

### CODE REVIEW

File: src/climax/climate_projection/arch.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+        # use a linear prediction head for this task
+        self.head = nn.Linear(embed_dim, img_size[0]*img_size[1])
+
+        if freeze_encoder:
+            for name, p in self.blocks.named_parameters():
+                name = name.lower()
+                if 'norm' in name:
+                    continue
+                else:
+                    p.requires_grad_(False)
```
Comment (rejuvyesh) [inline src/climax/climate_projection/arch.py:75]: Useful to add a shot comment/note here, pointing out that we don't freeze the norm layers.

File: src/climax/climate_projection/module.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
+# credits: https://github.com/ashleve/lightning-hydra-template/blob/main/src/models/mnist_module.py
```
Comment (rejuvyesh) [inline src/climax/climate_projection/module.py:1]: not really true for us anymore
```suggestion
```

**Review by rejuvyesh — CHANGES_REQUESTED** [review]
Small things, but otherwise looks great! Thanks for getting to it!

**Comment by tung-nd** [comment]
Thanks Jayesh! I just pushed to update the changes

**Review by rejuvyesh — APPROVED** [review]

**Pull request merged**